generator/cli/generate-migration-scripts.ts

```diff
@@ -1,6 +1,8 @@
+import { FileTransformers } from '../../src/fileTransformers'
 import { Reflector } from '@prisma-spectrum/reflector'
 import { PromisePool } from '@supercharge/promise-pool'
 import { PrismaTemplates } from '~/src'
```

Comment:
```suggestion
import type { PrismaTemplates } from '~/src'
```

To make this more clear. This will be automated soon with an aslant plugin. But for this PR it helps explain why this import is working while your new one is not (static versus runtime).